### PR TITLE
SEOImages: User Story 1463880, Part 15

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: 
https://contentqacats.azurewebsites.net/CheckResults/2019031600054/Report/

There are two image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

File: docs/framework/wcf/feature-details/routing-introduction.md [hunk 1]
```diff
@@ -351,19 +351,19 @@ rc.FilterTable.Add(new MatchAllMessageFilter(), backupList);
...
-|One-Way|![Check mark](media/checkmark.gif "Checkmark")|![Check mark](media/checkmark.gif "Checkmark")|![Check mark](media/checkmark.gif "Checkmark")|Yes|Abort the current transaction and create a new one. Resend all previous messages in the session. After a transaction has been created in which all messages have been successfully sent and the session indicates no more messages, all the outbound session channels are closed, receive contexts are all completed with the transaction, the inbound session channel is closed, and the transaction is committed.<br /><br /> When the sessions are being multicast the messages that had no error are resent to the same destination as before, and messages that encountered an error are sent to backup destinations.|  
+|One-Way||![Check mark](./media/routing-introduction/checkmark-indicates-compatibility.gif)||No|An exception is thrown and the transaction is rolled back.|
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/routing-introduction.md]: Let's remove this image and replace it with this character instead
```suggestion
|One-Way||✓||No|An exception is thrown and the transaction is rolled back.|  
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks great. Have comments for one of the articles.

**Comment by damabe** [comment]
@mairaw The changes are made. New CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031900041/Report/

There are two image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

**Comment by damabe** [comment]
@mairaw I got wires crossed while going between branches. I added three files to this PR. Sorry about that. It is this markdown file: https://review.docs.microsoft.com/en-us/dotnet/framework/wpf/graphics-multimedia/extend-glass-frame-into-a-wpf-application?branch=pr-en-us-11335 and the associated images.

The updated CATS report is here: https://contentqacats.azurewebsites.net/CheckResults/2019031900057/Report/

**Comment by mairaw** [comment]
No worries @damabe.

**Review by mairaw — APPROVED** [review]
LGTM 🆒

**Pull request merged**